frontend/src/components/layout/UserDropDown/UserDropDown.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Popper, Box, List, ListItem, ListItemText } from '@mui/material';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Switch from '@mui/material/Switch';
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
+
+import {
+  UserInfo,
+  UserAvatar,
+  UserName,
+  UserPopper,
+  ArrowDown,
+  ArrowUp,
+  DividerHorizontal,
+  Link,
+} from 'components/Layout/UserDropDown/styled';
+import { User } from 'types/user';
+
+interface PopperControl {
+  open: boolean;
+  onClick: (event: React.MouseEvent<HTMLElement>) => void;
+  anchorEl: HTMLElement | null;
+}
+
+type Props = User & PopperControl;
+
+const DropDown: React.FC<Props> = ({
+  userName,
+  userRole,
+  userUnit,
+  userDepartment,
+  userGroup,
+  avatarUrl,
+  open,
+  onClick,
+  anchorEl,
+}) => (
+  <Box>
+    <UserInfo onClick={(event: React.MouseEvent<HTMLElement>) => onClick(event)}>
+      <UserAvatar src={avatarUrl} />
+      <UserName>{userName}</UserName>
+      {open ? <ArrowUp /> : <ArrowDown />}
+    </UserInfo>
+    <Popper open={open} anchorEl={anchorEl}>
+      <UserPopper>
+        <FormControlLabel control={<Switch />} label="Autopass" />
+        <DividerHorizontal />
+        <List>
+          <ListItem disablePadding>
+            <ListItemText primary={'Role: ' + userRole} />
+          </ListItem>
+          <ListItem disablePadding>
+            <ListItemText primary={'Unit: ' + userUnit} />
+          </ListItem>
+          <ListItem disablePadding>
+            <ListItemText primary={'Department: ' + userDepartment} />
+          </ListItem>
+          <ListItem disablePadding>
+            <ListItemText primary={'Group: ' + userGroup} />
```

Comment:
Remove please all the calculations.
Replace them with:
`<ListItemText primary={`Group: ${userGroup}`} />`